Artifacts/MsSql/Structure/Security/0040-Tables-MultipleAuthStrats.sql

```diff
@@ -0,0 +1,131 @@
+-- SPDX-License-Identifier: Apache-2.0
+-- Licensed to the Ed-Fi Alliance under one or more agreements.
+-- The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+-- See the LICENSE and NOTICES files in the project root for more information.
+
+------------------------------ [dbo].[ResourceClaimActions] -------------------------------------------------
+CREATE TABLE [dbo].[ResourceClaimActions](
+	[ResourceClaimActionId] [int] IDENTITY(1,1) NOT NULL,
+	[ActionId] [int] NOT NULL,
+	[ResourceClaimId] [int] NOT NULL,
+	[ValidationRuleSetName] [nvarchar](255) NULL,
+ CONSTRAINT [PK_dbo.ResourceClaimActions] PRIMARY KEY CLUSTERED 
+(
+	[ResourceClaimActionId] ASC
+)WITH (PAD_INDEX = OFF, STATISTICS_NORECOMPUTE = OFF, IGNORE_DUP_KEY = OFF, ALLOW_ROW_LOCKS = ON, ALLOW_PAGE_LOCKS = ON) ON [PRIMARY]
+) ON [PRIMARY]
+GO
+
+-- NONCLUSTERED INDEX for [ActionId],[ResourceClaimId] columns 
+CREATE NONCLUSTERED INDEX [IX_ActionId_ResourceClaimId]
```

Comment:
This should be a unique constraint/index -- and the order of the columns in the index should still follow the order of columns shown above (i.e. `ResourceClaimId` then `ActionId`)

Reply:
I updated with CREATE UNIQUE NONCLUSTERED INDEX